packages/react-components/src/templates/NewsDetailsPage.tsx

```diff
@@ -51,7 +51,7 @@ const NewsDetailsPage: React.FC<NewsDetailsPageProps> = ({
   return (
     <div css={containerStyles}>
       <Card>
-        <Pill>{type}</Pill>
+        <Pill>{type ?? 'Tutorial'}</Pill>
```

Comment:
Just want to confirm that all news has a type

Reply:
I am going to double check, just a moment :)

Reply:
since before the changes in this PR we had just

```
<Pill>{type}</Pill>
```

I think we were assuming that News always had type